orchagent/flexcounterorch.cpp

```diff
@@ -328,11 +329,41 @@ bool FlexCounterOrch::bake()
     return consumer->addToSync(entries);
 }
 
+static bool isCreateOnlyConfigDbBuffers(Table& deviceMetadataConfigTable)
+{
+    std::string createOnlyConfigDbBuffersValue;
+
+    try
+    {
+        if (deviceMetadataConfigTable.hget("localhost", "create_only_config_db_buffers", createOnlyConfigDbBuffersValue))
```

Comment:
should those const strings be declared in header file with constexpr and capital letters ?

Reply:
I saw in the orchagent code, that it is a common practise to don't declare a dedicated const string variable if it is only used one time